docs/variants/novacustom_ns5x_adl/hardware-matrix.md

```diff
@@ -5,6 +5,16 @@
 This document describes the hardware configurations used for validation of the
 coreboot port on the NovaCustom NS5x/7x laptops.
 
+## Ports specification
+
+![](/images/novacustom_ns_ports_right_view.png)
```

Comment:
@Pokisiekk Instead of adding explanations on the graphic, couldn't we gather them in a table (e.g., with an additional column)?

The proposed approach is problematic in that the graphics will have to be updated with each change.

Reply:
Updated - https://github.com/Dasharo/docs/pull/590/commits/4013d16cbc7026def97a6c8e6b18dce47cc37a69